iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/RegistryManagerTests.java

```diff
@@ -459,35 +459,42 @@ public void apply_configuration_e2e() throws Exception
     public void deviceCreationWithDeviceScope() throws IOException, InterruptedException, IotHubException, URISyntaxException
     {
         // Arrange
-        RegistryManagerTestInstance testInstance = new RegistryManagerTestInstance();
-        deleteDeviceIfItExistsAlready(testInstance.registryManager, testInstance.deviceId);
+        String edge1Id = deviceIdPrefix + UUID.randomUUID().toString();
+        String edge2Id = deviceIdPrefix + UUID.randomUUID().toString();
+
+        RegistryManagerTestInstance deviceInstance = new RegistryManagerTestInstance();
```

Comment:
No need to create this instance. Everywhere you are currently using it can be replaced by "this.testInstance" to get the test instance that is made for you by the setup function for this test suite